doc/source/reference/release_2.0.rst

```diff
@@ -39,3 +39,66 @@ API changes
        # recommended
        >>> tree.minimum_spanning_tree(G, algorithm='kruskal', weight='mass')
        >>> tree.minimum_spanning_edges(G, algorithm='prim', weight='mass')
+
+* [`<>`_]
```

Comment:
Move this to the very top. This is probably the single most important change in 2.0.